bentoml/_internal/bento/docker.py

```diff
@@ -18,7 +18,13 @@
 # Python supported versions
 SUPPORTED_PYTHON_VERSIONS = ["3.7", "3.8", "3.9", "3.10"]
 # CUDA supported versions
-SUPPORTED_CUDA_VERSIONS = {"11": "11.6.2"}
+SUPPORTED_CUDA_VERSIONS = ["11.6.2"]
```

Comment:
made this a list of CUDA versions that can actually be installed on distro. The ALLOWED_CUDA_VERSION_ARGS provides a map from user provided version string to the actual version to be installed

Reply:
sg